### ISSUE TITLE
TEXT VETTED: 0630IbnCunaynShacir.Diwan.JK007597-ara1

### ISSUE THREAD

**Issue by hamidrh66** [OpenITI/Annotation#2279]
**#URI_CHECKED:#** y

**#EXTENTION_CHANGED:#** y

**#VERSION_YML:#** y: permalinks updated, issues updated

**#AUTHOR_YML_CHECKED:#** y: name transcribed, birth and death date added, EI2 link added

**#BOOK_YML_CHECKED:#** y: title transcribed

**#STRUCTURAL_ANNOTATION:#** See NOTE OF THE REVIEWER in the yml file below.

**#OUTSTANDING_FORMALIZED_ISSUES:#** NO_MAJOR_ISSUES

**#FURTHER_COMMENTS#** 

NB: Make sure the issue title looks like TEXT VETTED: /uri of the vetted text/.

More info on vetting: see https://docs.google.com/document/d/1twaKWlfiu92pLbFEF5JDRsgo9HQsuOZKSsO4uTjGP_g/edit?usp=sharing

**Comment by hamidrh66** [comment]
00#VERS#CLENGTH##: 73913
00#VERS#LENGTH###: 17975
00#VERS#URI######: 0630IbnCunaynShacir.Diwan.JK007597-ara1
80#VERS#BASED####: NO_DATA_IN_METADATA
80#VERS#COLLATED#: http://www.worldcat.org/oclc/4771257490
80#VERS#LINKS####: https://archive.org/details/a1007n
90#VERS#ANNOTATOR: Gowaart Van Den Bossche
90#VERS#COMMENT##: I have cleaned up the text and formatted all the poetry. I compared this version with the published version by Mardam Bek, but they are in fact not exactly the same. The digital edition only preserves the poems themselves, in the same order as the printed edition, but leaves out all the contextual information and the subtitles of the work. By contrast the digital edition provides the meter for every poem, which is not given by the edition. The material left out is in fact very valuable information for correctly interpreting the poems, so a new edition is needed. 
   NOTE OF THE REVIEWER (HRH): The difference between the PDF and the digital edition pointed out by the annotator is probably simply because the digital edition is based on an edition different from the edition of the PDF. It doesn’t seem logical that someone intentionally remove the contextual information and add the meter for poems. It seems more logical to be a different edition, especially because the page numbers in the PDF and the digital version are completely different. So, the text is complete as it is and I removed the MISSING_PARTS and NEW_EDITION_NEEDED labels added by the annotator. 2- There is nothing in the text that need to be tagged. However, the numbers of lines are situated at the end of previous lines. Maybe it’s not a bad idea to move them to the beginning of their own lines. This is not really a big deal, just to make the text of the best format possible. Maybe it’s possible to do it by RegEx.
90#VERS#DATE#####: 2020-05-07
90#VERS#ISSUES###: NO_MAJOR_ISSUES

**Comment by hamidrh66** [comment]
00#BOOK#URI######: 0630IbnCunaynShacir.Diwan
10#BOOK#GENRES###: src@keyword, src@keyword, src@keyword
10#BOOK#TITLEA#AR: Dīwān
10#BOOK#TITLEB#AR: Risālaŧ al-Muʾallif
20#BOOK#WROTE####: URIs from Althurayya, comma separated
30#BOOK#WROTE##AH: YEAR-MON-DA (X+ for unknown)
40#BOOK#RELATED##: URI of a book from OpenITI, or [Author's Title],
    followed by abbreviation for relation type between brackets (see
    book_relations repo). Only include relations with older books. Separate
    related books with semicolon.
80#BOOK#EDITIONS#: permalink, permalink, permalink
80#BOOK#LINKS####: permalink, permalink, permalink
80#BOOK#MSS######: permalink, permalink, permalink
80#BOOK#STUDIES##: permalink, permalink, permalink
80#BOOK#TRANSLAT#: permalink, permalink, permalink
90#BOOK#COMMENT##: a free running comment here; you can add as many
    lines as you see fit; the main goal of this comment section is to have a
    place to record valuable information, which is difficult to formalize
    into the above given categories.

**Comment by hamidrh66** [comment]
00#AUTH#URI######: 0630IbnCunaynShacir
10#AUTH#ISM####AR: Muḥammad
10#AUTH#KUNYA##AR: 
10#AUTH#LAQAB##AR: Šaraf al-dīn, Abū al-maḥāsin
10#AUTH#NASAB##AR: b. Naṣr b. ʿAlī b. Muḥammad b. Ġālib
10#AUTH#NISBA##AR: al-Anṣārī al-Dimašqī
10#AUTH#SHUHRA#AR: Ibn ʿUnayn
20#AUTH#BORN#####: DIMASHQ_363E335N_S
20#AUTH#DIED#####: DIMASHQ_363E335N_S
20#AUTH#RESIDED##: URIs from Althurayya, comma separated
20#AUTH#VISITED##: URIs from Althurayya, comma separated
30#AUTH#BORN###AH: 0549-SHC-09
30#AUTH#DIED###AH: 0630-RA1-20
40#AUTH#STUDENTS#: AUTH_URI from OpenITI, comma separated
40#AUTH#TEACHERS#: AUTH_URI from OpenITI, comma separated
80#AUTH#BIBLIO###: src@id, src@id, src@id, src@id, src@id
90#AUTH#COMMENT##: EI2: "Abu ’l-Maḥāsin S̲h̲araf al-Dīn Muḥ. b. Naṣr b. ʿAlī b. Muḥ. b. G̲h̲ālib al-Anṣārī , satirical poet born at Damascus on 9 S̲h̲aʿbān 549/19 October 1154, and died there on 20 Rabīʿ I 630/4 January 1233.:" http://dx.doi.org/10.1163/1573-3912_islam_SIM_3401

**Comment by hamidrh66** [comment]
https://github.com/OpenITI/Annotation/issues/2280

**Comment by pverkind** [comment]
Hi Hamid, I think Gowaart's point on the edition is that the digital text as we have it does not include important information between the poems, and that a digital edition that includes that information would be useful. 
I removed the line numbers rather than putting them on a new line. 

@Gowaart, do you think this additional information is original, or editorial?

**Comment by Gowaart** [comment]
Indeed. I don't know the manuscript tradition for this particular text, but having looked at the PDF edition, it would seem this additional information is original (i.e. info on why the poet wrote a particular poem, in response to some event etc.). So yes, new edition needed to better represent actual text tradition. This is the case for most of the diwans in our corpus. The diwans in our corpus were probaby compiled from databases, compilers simply put the poems in the right order as found in the editions but did not include material that was not in those databases (text introductions, introductions to particular poems, clarifications of terms, artful prose texts, etc.)

**Comment by pverkind** [comment]
OK, then I'll put the MISSING_PARTS back in!

**Comment by hamidrh66** [comment]
Hi Peter,
Thanks for checking this. I think the additional information in the PDF is original and of course very important. I just mean this version that we have on the corpus is probably a different edition from the edition of the PDF. Because page numbers are different. Of course it is a good idea to find and add the a digital version of the edition of the PDF, but I think we should add it as a new file and tag it and make it primary.
